org.jousse.vincent.Pomodorolm.yml

```diff
@@ -0,0 +1,83 @@
+id: org.jousse.vincent.Pomodorolm
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.node20
+
+command: pomodorolm
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --socket=pulseaudio # We want to be able to play sounds
+  - --device=dri # OpenGL
+  - --share=ipc
+  - --env=FLATPAK=1 # Used to tell the app to look for assets in /app/lib/pomodorolm
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Needed for the systray to work properly. libappindicator emits a dbus signal with the systray icon absolute path
+  # so flatpak and the host need to both have access to the icon for the systray to work properly
+  - --filesystem=xdg-data/tray-icon
```

Comment:
I don't understand why this is still needed. `$XDG_DATA_HOME` inside the Flatpak sandbox is set to `~/.var/app/$FLATPAK_ID/data`.

So not having this permission will mean the tray icon goes in `~/.var/app/$FLATPAK_ID/data` which is a path accessible on host.

Why do you want it on host's $XDG_DATA_HOME which is usually ~/.local/share/tray-icon` ?

Reply:
Indeed, it's not needed anymore, I've removed it